docs/wallets/mainnet-address-generation.md

```diff
@@ -0,0 +1,242 @@
+# Mainnet Address Generation
+
+This document contains the information of account address for Crypto.org Chain mainnet.
+
+## Address prefix
+
+Account address for mainnet start with prefix `cro`. For example: `cro1y8ua5laceufhqtwzyhahq0qk7rm87hhugtsfey`.
+
+## Hierarchical Deterministic Wallet (HD Wallet) Derivation Path
+
+Crypto.org Chain has [registered](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) coin type `394` as defined in [BIP44 standard](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki).
+
+Coin Type: 394 <br />
+Derivation Path: `44'/394'/0'/0/{index}`  where index starts from 0.
+
+i.e. the first address is derived at path `44'/394'/0'/0/0` and second one at path `44'/394'/0'/0/1`.
+
+## How to generate address
+
+::: warning MAKE SURE YOU BACKUP YOUR MNEMONIC WORDS
+Before you use the generated mainnet addresses to receive funds on mainnet, make sure that you have backup your address's mnemonic words in a safe place and the mnemonic words are correct such that it can restore to your wallet address.
+
+You are the sole owner of your wallet mnemonic words and there is no way for Crypto.org to restore you wallet or recover you funds if you lost the mnemonic words.
+:::
+
+::: warning GENERATE ADDRESS ONLY IN A SECURE ENVIRONMENT
+We recommend you to generate an address only on a trusted and secure computer. To further enhance the security, you should consider to run on a air-gapped (offline) machine.
+:::
+
+::: warning ALWAYS VERIFY YOUR MNEMONIC WORDS AND ADDRESS
+To make sure you have backed up the mnemonic words correctly, we recommend you to try to restore your wallet with the mnemonic words and verify the address derived is the same.
+
+For more details on how to verify your mnemonic words and addresses, please check [Mainnet Address Verification](./mainnet-address-verification.md)
+:::
+
+## A. Release Binary (CLI)
+
+Supported OS: Linux, Mac OS and Windows
+
+#### Step 1. Get the Crypto.org Chain binary
+
+Download the Crypto.org Chain Binary from Crossfire from [release page](https://github.com/crypto-org-chain/chain-main/releases/tag/v0.9.1-crossfire) and extract the binary. Here we used Linux as an example:
+
+```bash
+$ curl -LOJ https://github.com/crypto-org-chain/chain-main/releases/download/v0.9.1-crossfire/chain-main_0.9.1-crossfire_Linux_x86_64.tar.gz
+$ tar -zxvf chain-main_0.9.1-crossfire_Linux_x86_64.tar.gz
```

Comment:
use 1.0.0 -- or will this be blocked by the log level issue mentioned on Slack?

Reply:
It was previously on hold because of it. But now it's resolved I will update to 1.0.0, thanks!

Reply:
Updated